src/datarobotx/idp/llm_blueprints.py

```diff
@@ -103,16 +113,38 @@ def get_or_register_llm_blueprint_custom_model_version(
         The DataRobot API token to use.
     llm_blueprint_id : str
         The ID of the LLM blueprint to use.
+    guard_kwargs : Optional[List[Dict[str, Any]]
+        Additional arguments to configure LLM guards.
+        An example format of a guard is:
+        {
+            "guard_config_template_name": "Prompt Injection",
+            "guard_config_template_settings": {
+                "deploymentId": "some_deployment_id",
+            },
+            "stages": ["prompt", "response"],
+            "intervention": {
+                "action": "block",
+                "conditions": [
+                    {
+                        "comparand": 0.5,
+                        "comparator": "greaterThan",
+                    }
+                ],
+            },
+        }
     **kwargs : Any
```

Comment:
suggest we explicitly pull out either the register_custom_model_kwargs or the create_from_pervious_kwargs into a separate explicit optioinal keyword argument to avoid any ambiguity; the one you don't pull out can be the **kwargs (probably the register_custom_model additional kwargs by precedent); i had to skim below to understand which arguments go where and it looks like we're manually having to pop them vs. just exploding the whole dict in